proto/buf/alpha/registry/v1alpha1/organization.proto

```diff
@@ -205,6 +205,12 @@ message UpdateOrganizationSettingsRequest {
   PluginRole plugin_base_role = 3;
   // optional, no update to this base role will be made if this is unspecified.
   TemplateRole template_base_role = 4;
+  // optional, no update to description will be made if this is unspecified.
+  string description = 5;
+  // optional, no update to url will be made if this is unspecified.
+  string url = 6;
+  // optional, no update to verification_status will be made if this is unspecified.
+  VerificationStatus verification_status = 7;
```

Comment:
I think it will be better to make this a separated endpoint for server admin only, as we are not going to let user update their own verification status

Reply:
You're right, should that be a public API then?